app/screens/find_channels/utils.ts

```diff
@@ -25,3 +25,18 @@ export const retrieveChannels = (database: Database, myChannels: MyChannelModel[
 
     return of$([]);
 };
+
+export const removeChannelsFromArchivedTeams = (recentChannels: ChannelModel[], teamIds: Observable<Set<string>>) => {
```

Comment:
If we receive directly the value instead of the observable, this function gets way simpler. 0/5 on whether the function should return an observable (`return of$(x)`) or just the value, and let the caller create the observable.